### PR TITLE
cpanel: point to Redis cluster on AWS EC

### PR DESCRIPTION
**Pull request by xoen**
Actually not, [these values will be ignored by current/old version of helm chart](https://github.com/ministryofjustice/analytics-platform-helm-charts/blob/414d5e7aae2db8ea9b10dfd803f9abc53f4d1a13/charts/cpanel/templates/env-configmap.yaml#L8-L16) so this is safe to merge even before helm chart change is merged.

~This depends on the followin changes~:
- [X] ~Redis Cluster provisioned: https://github.com/ministryofjustice/analytics-platform-ops/pull/368~
- [X] ~CP addition of `REDIS_SCHEME`: https://github.com/ministryofjustice/analytics-platform-control-panel/pull/843~
- [ ] ~CP helm chart setting of `REDIS_SCHEME`: https://github.com/ministryofjustice/analytics-platform-helm-charts/pull/474~

Part of ticket: https://trello.com/c/YG9anK8A

### CODE REVIEW

**Comment by xoen** [comment]
FYI: This should not depend on helm chart changes and we should be able to merge safely before that change (which is nice) as these values will be currently be ignored - and continue to connect to redis on k8s until new helm chart version is deployed.

TL;DR: Once this is approved we can merge without worrying to break things.

**Review by andylightfoot — APPROVED** [review]
yolo

**Comment by xoen** [comment]
Cheers Andy ☺️
